### ISSUE TITLE
SAM Snow Model with Ground-Mount PV System

### ISSUE THREAD

**Issue by shelbieldavis** [NREL/SAM#1547]
**Is your feature request related to a problem? Please describe.**
I would like to run a snow simulation (Marion Model) in SAM for a ground-mount PV system.

**Describe the solution you'd like**
The snow model in SAM uses 1.97 as the snow slide amount coefficient. For a ground mount, Marion suggests 6 as the snow slide amount coefficient. I would like the option to use 6 instead of 1.97. 

### Describe alternatives you've considered

Used pvlib as alternative.

### Additional context

See page 2-3 for SAM documentation on the Marion Model: https://www.nrel.gov/docs/fy17osti/68705.pdf

**Comment by mjprilliman** [comment]
Hi @shelbieldavis, thank you for submitting this issue. Our plan is to allow the user to change this sliding coefficient for the Marion snow model for use in the SAM SDK and PySAM. This change should be implemented for the first patch to the upcoming SAM release.

**Comment by shelbieldavis** [comment]
@mjprilliman Thank you for addressing this issue. I see it has been resolved, but I am not sure how to change the snow slide coefficient in a PySAM PVWatts no financial configuration. What is the name of the variable and can it be assigned in a PySAM configuration like tilt or array type? I tried to do this, but the configuration did not recognize my guesses at the variable name, e.g. "snow_slide_coeff" or "snow_slide_coefficient."

Example of variable assignment with array_type:
old_array_type = system_model.SystemDesign.array_type
new_array_type = 0

system_design_group_object = getattr(system_model,'SystemDesign')
setattr(system_design_group_object, 'array_type', new_array_type )
